docs/3_about.rst

```diff
@@ -57,6 +60,26 @@ License
 
 
 
+Alternative python packages
+---------------------------
+
+- `tzfpy <https://github.com/ringsaturn/tzfpy>`__ (less accurate, more lightweight, faster)
+- `pytzwhere <https://pypi.python.org/pypi/tzwhere>`__ (not maintained)
+
+
+Comparison to tzfpy
+-----------------------
+
+**Differences:**
+
+- ``tzfpy`` is a Python binding of the Rust package ``tzf-rs``
+- ``tzfpy`` has no startup time
+- ``tzfpy`` uses a hierarchical tree of rectangles as timezone polygons (data):
+    - this reduces the memory requirements
+    - this reduces the accuracy
+    - this increases the lookup speed
```

Comment:
This part has a little problem.

tzfpy uses QuadTree based index just for quick look up, like shortcuts in TimezoneFinder. If no data found in index, it will callback to tradition point-in-polygon search.

Accuracy is a little bit lower because polygon data is simplified, not because of using QuadTree based index.